packages/cli/src/agentInstaller/pythonAgentInstaller.js

```diff
@@ -76,15 +78,18 @@ class PipInstaller extends BuildToolInstaller {
    */
   get assumptions() {
     return `Your project contains a ${REQUIREMENTS_FILE}. Therefore, it looks like a pip project,
-    so we will add a dependency on the "appmap" package.`;
+so we will add a dependency on the "appmap" package.`;
   }
 
   /**
    * @returns {string}
    */
   get postInstallMessage() {
     return `The AppMap Python package ("appmap") has been added to your ${REQUIREMENTS_FILE}. You should open this file and check that
-    it looks clean and correct.`;
+it looks clean and correct.
+    
+Once you've done that, you'll complete the installation of the AppMap package by running the following command
```

Comment:
This is the step where it didn’t sound totally clear that pushing enter was going to run the confirmation command.

Reply:
Maybe instead of “once you’ve done that”, simply “Now…”. Or maybe we should show them a diff of the file?